foyer/forcefields/ff.xsd

```diff
@@ -185,6 +222,16 @@
       </xs:choice>
     </xs:complexType>
   </xs:element>
+  <xs:element name="CustomTorsionForce">
+    <xs:complexType>
+      <xs:choice maxOccurs="unbounded">
+        <xs:element ref="Proper" minOccurs="0"/>
+        <xs:element ref="PerTorsionParameter" minOccurs="0"/>
+        <xs:element ref="Improper" minOccurs="0"/>
+      </xs:choice>
+      <xs:attribute type="xs:string" name="energy" use="optional"/>
```

Comment:
Does `CustomTorsionForce` not require an energy definition?

Reply:
Yes, looks like it's a required attribute